.ci/magic-modules/release-ansible.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+# This script takes in 'magic-modules-branched', a git repo tracking the head of a PR against magic-modules.
+# It outputs "ansible-generated", a non-submodule git repo containing the generated ansible code.
+
+set -x
+set -e
+
+pushd magic-modules-gcp
+./tools/ansible-pr/run
```

Comment:
Probably not the right way to do this - couldn't you run the script directly from the yaml file?